### ISSUE TITLE
`syncrypt pull` prints, that it ignored an exception?

### ISSUE THREAD

**Issue by dersmon** [syncrypt/client#2]
I successfully uploaded an image, but when I try to pull in the same folder, this happens:

```
$ syncrypt pull
INFO:syncrypt.backends.binary:Getting a list of changes for <Vault: .> (latest to efca8e56-c7c0-49ed-8533-e55e2f728172)
Exception ignored in: $
```

**Comment by lordi** [comment]
Thats interesting, I can't reproduce it here. Are you running python 3.5 or python 3.4?

**Comment by lordi** [comment]
I added a workaround for Py 3.4 in 6df0fd30d3ece8b5cd87db277572f3ce1cd80747
Can you try the latest version v0.1.4?

**Comment by lordi** [comment]
I assume that it is fixed in 0.1.4, please re-open if that is not the case. Thx :)
